components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestions.kt

```diff
@@ -37,6 +38,7 @@ internal fun Suggestions(
         modifier = Modifier.testTag("mozac.awesomebar.suggestions")
     ) {
         val suggestions = fetcher.state.value
+        val showEditSuggestion: Boolean = showEditSuggestions()
```

Comment:
Added this so we don't call the function every time below.

Reply:
(See my comment above. Definitely good to not call it for every suggestion, but I don't even see why we need to call it dynamically on a Suggestion level. Maybe I am missing a feature you're thinking of.)

Reply:
In this case I'm not calling for every suggestion.  I added this so every suggestion will use the same value set on this line.

Reply:
I know, like I wrote above, but you're calling it for every set of suggestions which seems unnecessary.